### ISSUE TITLE
Update docker container versions

### ISSUE THREAD

**Issue by docktermj** [Senzing/aws-cloudformation-ecs-poc-simple#98]
Based on the current values in the list at https://github.com/Senzing/knowledge-base/blob/29a30df522d8bc727982bd10ea24f690d681a879/lists/versions-latest.sh,  The following docker images can be updated:

1. `public.ecr.aws/senzing/senzing-api-server:2.6.1`
1. `public.ecr.aws/senzing/jupyter:1.3.0`
1. `public.ecr.aws/senzing/redoer:1.3.7`
1. `public.ecr.aws/senzing/sshd:1.2.0`
1. `public.ecr.aws/senzing/stream-loader:1.7.6`
1. `public.ecr.aws/senzing/swagger-ui:v3.50.0`

Because `stream-producer` and `stream-loader` access new resources, the following needs to be added:

In IamPolicySqsConsumer and IamPolicySqsConsumerRedo:
 - Add `sqs:GetQueueAttributes`

Other updates based on work in https://github.com/Senzing/aws-cloudformation-ecs

1.  In EcsTaskDefinitionRedoerLoader:

    ```yaml
              Name: SENZING_REDO_SLEEP_TIME_IN_SECONDS
              Value: '10'
    ```

1. In EcsTaskDefinitionRedoerProducer:

    ```yaml
            -
              Name: SENZING_REDO_SLEEP_TIME_IN_SECONDS
              Value: '10'
              :
            -
              Name: SENZING_THREADS_PER_PROCESS
              Value: '12'
    ```

1. Once the new Senzing version is available

    ```yaml
      Version: senzingapi-2.7.0-00000
    ```

Once this is done, we can make a `1.1.0` versioned release.

**Comment by roncewind** [comment]
actual version:  2.7.0-21153

**Comment by roncewind** [comment]
PR merged.

**Comment by roncewind** [comment]
The redoer got reverted to 1.3.5

**Comment by roncewind** [comment]
sshd also got reverted.

**Comment by roncewind** [comment]
new PR merged
